products/quasar.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Quasar
+category: framework
+tags: javascript-runtime
+iconSlug: quasar
+permalink: /quasar
+versionCommand: quasar -v
+releasePolicyLink: https://github.com/quasarframework/quasar/blob/dev/ROADMAP.md#support-policy-and-schedule
+changelogTemplate: https://github.com/quasarframework/quasar/releases/tag/quasar-v__LATEST__
+activeSupportColumn: true
+releaseDateColumn: true
+
+auto:
+-   npm: quasar
+
+identifiers:
+-   purl: pkg:npm/quasar
+
+releases:
+-   releaseCycle: "2"
+    support: true
+    eol: false
+    latest: "2.12.7"
+    lts: false
+    latestReleaseDate: 2023-09-13
+    releaseDate: 2021-07-21
+
+-   releaseCycle: "1"
+    support: 2021-04-01
+    eol: 2023-06-30
+    latest: "1.22.10"
+    lts: false
+    latestReleaseDate: 2023-08-09
+    releaseDate: 2019-03-07
+
+---
+
+> [Quasar](https://quasar.dev/)  is an MIT licensed open-source Vue.js based framework, which allows you as a web developer to quickly create responsive++ websites/apps in many flavours.
+
+All major releases are typically supported for at least 12 months. LTS support ends 12 months after the release of a new major version.
```

Comment:
Would be great to explain if possible what implies active support and what implies LTS support (for example LTS support => security patch only).

Reply:
Did not found anything about that, let's forget it !